operations/model.go

```diff
@@ -338,24 +347,28 @@ func (s *ClientSettings) SetDefaultAlias(project string, alias string) {
 	})
 }
 
-func (s *ClientSettings) SetDefaultProject(name string) {
-	var foundDefault bool
-	for i, p := range s.Projects {
-		if p.Name == name {
-			s.Projects[i].Default = true
-			foundDefault = true
-		} else {
-			s.Projects[i].Default = false
-		}
+func (s *ClientSettings) SetDefaultProject(project string) {
+	if s.DontSetDefaultProject {
+		return
+	}
+	cwd, err := os.Getwd()
+	if err != nil {
+		grip.Error(errors.Wrap(err, "unable to get current working directory"))
+		return
+	}
+	cwd, err = filepath.EvalSymlinks(cwd)
```

Comment:
Can loadProject pass in cwd so you don't have to do this twice?

Reply:
done